tests/integration/nfs_test.go

```diff
@@ -0,0 +1,147 @@
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+// *******************************************************
+// *** Major scenarios tested by the NfsSuite ***
+// Setup
+// - via the server CRD with very simple properties
+//   - 1 replica
+//   - Default server permissions
+//   - Mount a NFS export and write data to it and verify
+// *******************************************************
+func TestNfsSuite(t *testing.T) {
+	s := new(NfsSuite)
+	defer func(s *NfsSuite) {
+		HandlePanics(recover(), s, s.T)
+	}(s)
+	suite.Run(t, s)
+}
+
+type NfsSuite struct {
+	suite.Suite
+	k8sHelper       *utils.K8sHelper
+	installHelper   *installer.InstallHelper
+	rwClient        *clients.ReadWriteOperation
+	namespace       string
+	systemNamespace string
+	instanceCount   int
+}
+
+func (suite *NfsSuite) SetupSuite() {
+	suite.SetUp()
+}
+
+func (suite *NfsSuite) TearDownSuite() {
+	suite.TearDown()
+}
+
+func (suite *NfsSuite) SetUp() {
+	suite.namespace = "nfs-ns"
+	suite.systemNamespace = installer.SystemNamespace(suite.namespace)
+	suite.instanceCount = 1
+
+	k8sHelper, err := utils.CreateK8sHelper(suite.T)
+	require.NoError(suite.T(), err)
+	suite.k8sHelper = k8sHelper
+
+	k8sversion := suite.k8sHelper.GetK8sServerVersion()
+	logger.Infof("Installing nfs server on k8s %s", k8sversion)
+
+	suite.installHelper = installer.NewK8sRookhelper(suite.k8sHelper.Clientset, suite.T)
+
+	suite.rwClient = clients.CreateReadWriteOperation(suite.k8sHelper)
+
+	err = suite.installHelper.InstallNFSServer(suite.systemNamespace, suite.namespace, suite.instanceCount)
+	if err != nil {
+		logger.Errorf("nfs server installation failed: %+v", err)
+		suite.T().Fail()
+		suite.TearDown()
+		suite.T().FailNow()
+	}
+}
+
+func (suite *NfsSuite) TearDown() {
+	if suite.T().Failed() {
+		installer.GatherNFSServerDebuggingInfo(suite.k8sHelper, suite.systemNamespace)
+		installer.GatherNFSServerDebuggingInfo(suite.k8sHelper, suite.namespace)
+		suite.installHelper.GatherAllNFSServerLogs(suite.systemNamespace, suite.namespace, suite.T().Name())
+	}
+	suite.installHelper.UninstallNFSServer(suite.systemNamespace, suite.namespace)
+}
+
+func (suite *NfsSuite) TestNfsServerInstallation() {
+	logger.Infof("Verifying that nfs server pod %s is running", suite.namespace)
+
+	// verify nfs server operator is running OK
+	assert.True(suite.T(), suite.k8sHelper.CheckPodCountAndState("rook-nfs-operator", suite.systemNamespace, 1, "Running"),
+		"1 rook-nfs-operator must be in Running state")
+
+	// verify nfs server instances are running OK
+	assert.True(suite.T(), suite.k8sHelper.CheckPodCountAndState("rook-nfs", suite.namespace, suite.instanceCount, "Running"),
+		fmt.Sprintf("%d rook-nfs pods must be in Running state", suite.instanceCount))
+
+	// determine the nfs operator pod name
+	podNames, err := suite.k8sHelper.GetPodNamesForApp("rook-nfs-operator", suite.systemNamespace)
```

Comment:
this operator pod name doesn't get used for anything useful.  it was useful in the CockroachDB test because we exec to it and perform SQL operations, but we don't use it in this NFS test.  I think you can remove it.